### PR TITLE
Validate IP Address against regex, move all ip validations back to schema

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/components/clusterConfiguration/ClusterConfiguration.tsx [hunk 1]
```diff
@@ -97,6 +98,13 @@ const validateVIP = (
   values: ClusterConfigurationValues,
   value: string,
 ) => {
+  if (!value) {
```
Comment (mareklibra) [inline src/components/clusterConfiguration/ClusterConfiguration.tsx:101]: Is there a reason why we don't use validation schemas here?

File: src/components/ui/formik/validationSchemas.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as Yup from 'yup';
...
+export const IP_ADDRESS_REGEX = /^(((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]))|([0-9a-f]{1,4}:){7,7}[0-9a-f]{1,4}|([0-9a-f]{1,4}:){1,7}:|([0-9a-f]{1,4}:){1,6}:[0-9a-f]{1,4}|([0-9a-f]{1,4}:){1,5}(:[0-9a-f]{1,4}){1,2}|([0-9a-f]{1,4}:){1,4}(:[0-9a-f]{1,4}){1,3}|([0-9a-f]{1,4}:){1,3}(:[0-9a-f]{1,4}){1,4}|([0-9a-f]{1,4}:){1,2}(:[0-9a-f]{1,4}){1,5}|[0-9a-f]{1,4}:((:[0-9a-f]{1,4}){1,6})|:((:[0-9a-f]{1,4}){1,7}|:)|fe80:(:[0-9a-f]{0,4}){0,4}%[0-9a-z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))$/;
```
Comment (mareklibra) [inline src/components/ui/formik/validationSchemas.ts:6]: Already wrapped by `ipValidationSchema()`, if needed

**Comment by rawagner** [comment]
@mareklibra regex way was just a quick fix for @jtomasek . I've updated the PR using the Yup validation, ptal

File: src/components/ui/formik/validationSchemas.ts (hunk 2 above)
Comment (andybraren) [reply src/components/ui/formik/validationSchemas.ts]: `// Future bug-fixer: Beer on me! (mlibra)`
😂

File: src/components/clusterConfiguration/ClusterConfiguration.tsx [hunk 3]
```diff
@@ -217,7 +204,11 @@ const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster })
   return (
     <Formik
       initialValues={initialValues}
-      validationSchema={submitType === 'install' ? installValidationSchema : validationSchema}
+      validationSchema={
```
Comment (mareklibra) [inline src/components/clusterConfiguration/ClusterConfiguration.tsx:207]: I think this should be memoized

**Pull request merged**